packages/config/config/devices/0x0234/zif5028_heatit_z-din_616.json

```
@@ -314,7 +485,7 @@
 			"defaultValue": 0,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": false,
+			"allowManualEntry": true,
 			"options": [
 				{
 					"label": "Output is controlled via Z-Wave messages.",
```

```suggestion
					"label": "Controlled via Z-Wave messages",
```
and the next: "Controlled by input 1"

---

i dont understand the "allowManualEntry" set to "true" when there's an option ?